buildSrc/src/main/kotlin/elastic-otel.agent-packaging-conventions.gradle.kts

```diff
@@ -182,6 +182,9 @@ tasks {
     dependsOn(isolateJavaagentLibs)
     configurations = listOf(bootstrapLibs, upstreamAgent)
 
+    // exclude slf4j-simple from the shadow jar as we use log4j2-slf4j with internal-logging instead
+    exclude("inst/io/opentelemetry/javaagent/slf4j/simple/**")
```

Comment:
What happens if OTEL_JAVAAGENT_LOGGING is set to simple?

Reply:
Because the implementation is not present, a noop internal logger is used. I think we could maybe transparently convert `simple` with `elastic` in the case someone is taking default configuration and making it explicit to prevent unexpected changes in defaults.

Reply:
Trying to set `OTEL_JAVAAGENT_LOGGING=simple` does produce an ugly SLF4J error:

```
SLF4J(E): A service provider failed to instantiate:
io.opentelemetry.javaagent.slf4j.spi.SLF4JServiceProvider: Provider io.opentelemetry.javaagent.slf4j.simple.SimpleServiceProvider not found
SLF4J(W): No SLF4J providers were found.
SLF4J(W): Defaulting to no-operation (NOP) logger implementation
```

So I've changed the behavior in https://github.com/elastic/elastic-otel-java/pull/613/commits/0b05c841bb2eb1710310bf23545af3431b491644, now when it's explicitly set to `simple` we still override the provided value and use the `elastic` logger.